homeassistant/components/mqtt/mixins.py

```diff
@@ -811,7 +811,8 @@ async def _subscribe_topics(self):
     @property
     def entity_registry_enabled_default(self) -> bool:
         """Return if the entity should be enabled when first added to the entity registry."""
-        return self._config[CONF_ENABLED_BY_DEFAULT]
```

Comment:
a `scene` cannot be disabled and will not have `CONF_ENABLED_BY_DEFAULT` as configuration option, despite if what the current documentation says. The PR to correct this is attached.

Reply:
Why can't scene entities be disabled?

Reply:
my bad, CONF_ENABLED_BY_DEFAULT added to the schema. This option is removed from the current documentation, because it is not available in the current code.